waterlog/src/Globals.js

```diff
@@ -1,3 +1,4 @@
 export const Globals = {
-	API_URL: 'api.iot.retrotest.co.za'
+	//API_URL: 'https://api.iot.retrotest.co.za'
+	API_URL: 'https://localhost:44382'
```

Comment:
Pretty sure you need the other URL and not localhost when pushing to the repo

Reply:
yes i've changed it